identity/conf/routes

```diff
@@ -52,3 +41,23 @@ OPTIONS    /email
 GET         /delete                                 controllers.AccountDeletionController.renderAccountDeletionForm
 POST        /delete                                 controllers.AccountDeletionController.processAccountDeletionForm
 GET         /delete/confirm                         controllers.AccountDeletionController.renderAccountDeletionConfirmation(autoDeletion: Boolean)
+
+########################################################################################################################
+# EDIT PROFILE
+########################################################################################################################
+GET         /public/edit                            controllers.EditProfileController.displayPublicProfileForm
+POST        /public/edit                            controllers.EditProfileController.submitPublicProfileForm
+
+GET         /account/edit                           controllers.EditProfileController.displayAccountForm
+POST        /account/edit                           controllers.EditProfileController.submitAccountForm
+
+GET         /membership/edit                        controllers.EditProfileController.displayMembershipForm
+GET         /contribution/recurring/edit            controllers.EditProfileController.displayRecurringContributionForm
+GET         /digitalpack/edit                       controllers.EditProfileController.displayDigitalPackForm
+
+GET         /privacy/edit                           controllers.EditProfileController.displayPrivacyForm
+POST        /privacy/edit                           controllers.EditProfileController.submitPrivacyForm
+
+GET         /email-prefs                            controllers.EditProfileController.displayPrivacyForm
```

Comment:
i don't like the idea of serving an identical page under two different endpoints, can we turn one of these into an http redirect?